### DISCUSSION TITLE
Portfolio

### DISCUSSION THREAD

**Discussion by eusoumarl** [community/community#65130] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Boa noite, amigos. Tudo bem?
Preciso lançar um portfolio online e gostei muito dessa versão do site: https://sernadesigns.github.io/simplefolio-ember/
Quero aproveitar aqui e parabeniza os desenvolvedores pela iniciativa de disponibilizar uma versão gratuita do projeto. Gratidão.
Precisaria de uma ajuda. Sabe se alguém não esta disposto de subir uma versão funcional no meu git e configurar um portifólio (https://marcosarl.github.io/) pra mim. Eu não não sou da área de TI. Estou batendo a cabeça não consigo configurar. Alguém consegue me ajudar? Eu pago a hora do desenvolver.
Grato 
Marcos

**Comment by ggdio** [comment]
Opa amigo, consigo te ajudar sim, subi um teste no meu github se quiser dar uma olhada: https://ggdio.github.io/portfolio-template/

Você já preencheu os arquivos com seus dados para aparecer no site/portfolio ? 

Me avisa nos comentários quando tiver com os arquivos preenchidos com os dados, que eu te ajudo à publicar no github pages.

**Reply by eusoumarl** [reply]
Opa, meu amigo. Agradeço o retorno. Visualizei o teste que você fez e deu certo. O meu não fica assim. Fica aquela página de configuração. Eu inseri as minhas informações, mas fui olhar agora e não gravou. Eu salvei as informações e não está gravando. Vou baixar o projeto e subir novamente agora de manhã e te aviso. Muito obrigado pela ajuda. Vai me ajudar muito.

**Reply by eusoumarl** [reply]
Opa, ggdio, eu subi outro projeto com os dados para aparecer no portfolio. Fiz outro projeto com as informações: https://marcosarl.github.io/marl/. Eu só não consegui subir as imagens da foto do perfil e do projeto porque eu não localizei a pasta assets/images. De resto está ok.

Obrigado novamente.
Marcos

**Reply by ggdio** [reply]
Faz o seguinte, cria um repositório novo com um nome tipo "teste" e me dá acesso pra realizar commit e push, que eu coloco pra rodar pra vc. Se funcionar a gente coloca no seu repositório principal

**Reply by eusoumarl** [reply]
Fechado. Mas ao invés de criar outro repositório, eu te dei acesso a esse repositório: https://marcosarl.github.io/marl/

**Reply by ggdio** [reply]
Blz, chegando em casa já mexo lá pra vc

**Reply by ggdio** [reply]
Cara, acho que vc mexeu nas configurações desse repositório para usar a branch MASTER como github-pages, por favor volta essa configuração para o padrão que é a branch "gh-pages", senão ele vai ler o arquivo README.md

**Reply by eusoumarl** [reply]
Eu dei uma olhada e já está como gh-pages. Agora não sei se é no mesmo local que tu falou.

**Reply by ggdio** [reply]
Coloca meu acesso como administrador daquele repositório "marl" que eu dou uma olhada, mas fica em settings>pages>branch

**Reply by ggdio** [reply]
Cara, deleta essa informação de usuário e senha, não é seguro.
Faz um favor, cria um repositório em branco novo do zero no seu usuário, e me adiciona, que eu configuro do zero. Pois estes que vc criou vc modificou as configurações e não está funcionando como deveria.
Igual falei, tem que mudar o settings>pages e mudar de Master pra gh-pages

**Reply by eusoumarl** [reply]
ok. Cheguei em casa, deletei aqui. Vou mudar os dados e criar o novo repositório e já te falo. Vlw

**Reply by eusoumarl** [reply]
Criei marcos-lima e já te adicionai

**Reply by ggdio** [reply]
Consegui fazer funcionar no repo "marl", da uma olhada: https://marcosarl.github.io/marl/

**Reply by eusoumarl** [reply]
Funcionou aqui.... agora eu só precisa atualizar as informações e adicionar as fotos, ok?

**Reply by ggdio** [reply, marked as answer]
Cara mais facil me mandar mensagem no meu discord: masterdio

Adiciona la e falamos por lá blz ?

Marca a resposta como correta pra fechar essa thread, vlw

**Reply by eusoumarl** [reply]
Combinado

**Comment by eusoumarl** [comment]
Opa, brother. Estou chegando em casa tbm. E te dei acesso ao outro portifolio.Enviado do meu iPhoneEm 28 de ago. de 2023, à(s) 19:04, Guilherme Dio ***@***.***> escreveu:﻿
Opa, como vc criou um "fork" do meu repositório parece estar dando conflito na hora de publicar.
Neste caso, crie um repositório novo do zero, sem ser fork, e me adiciona como contribuidor.

**Comment by eusoumarl** [comment]
Opa, chegando em casa tbm eu te dei acesso ao outro repositório.Enviado do meu iPhoneEm 28 de ago. de 2023, à(s) 19:04, Guilherme Dio ***@***.***> escreveu:﻿
Opa, como vc criou um "fork" do meu repositório parece estar dando conflito na hora de publicar.
Neste caso, crie um repositório novo do zero, sem ser fork, e me adiciona como contribuidor.

**Comment by Fodase4** [comment]
Alguém poderia explicar?
